From: Ole Streicher <olebole@debian.org>
Date: Fri, 14 Oct 2016 21:33:02 +0200
Subject: Fix executable installation issue

This avoids the following error:

-- Installing: /build/purify-2.0.0/debian/purify/usr/share/cmake/purify/PurifyTargets-none.cmake
CMake Error at cpp/cmake_install.cmake:42 (file):
  file INSTALL cannot find
  "/build/purify-2.0.0/obj-x86_64-linux-gnu/cpp//build/purify-2.0.0/obj-x86_64-linux-gnu/purify".
Call Stack (most recent call first):
  cmake_install.cmake:63 (include)
---
 cpp/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpp/CMakeLists.txt b/cpp/CMakeLists.txt
index 6544533..a0a8569 100644
--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -36,7 +36,7 @@ endif()
 if(TARGET casacore::ms)
   add_executable(purify main.cc cmdl.cc AlgorithmUpdate.cc)
   target_link_libraries(purify libpurify)
-  set_target_properties(purify PROPERTIES OUTPUT_NAME ${PROJECT_BINARY_DIR}/purify)
+  set_target_properties(purify PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
 
   install(TARGETS purify
     EXPORT PurifyTargets
